src/features/pageObjects/reducers/addPageObject.thunk.js

```diff
@@ -48,11 +65,11 @@ export const addPageObjReducer = (builder) => {
         id,
         name,
         url,
-        library: lastSelectedLibrary || defaultLibrary,
+        library: lastSelectedLibrary,
         pathname,
         search,
         origin,
-        ...(lastSelectedLocatorType ? { locatorType: lastSelectedLocatorType } : {}),
+        locatorType: lastSelectedLocatorType,
```

Comment:
according to our types, locatorType is optional, no need to store default value here

Reply:
fixed